Sources/Core/AWSClientRuntime/Auth/CredentialsProviders/SSOCredentialsProvider.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+import ClientRuntime
+import Foundation
```

Comment:
nit: I don't think importing Foundation is needed here.  If not, remove the import